atc/db/dbtest/builder.go

```diff
@@ -291,7 +291,7 @@ func (builder Builder) WithPendingJobBuild(assign *db.Build, jobName string) Set
 			return fmt.Errorf("job '%s' not configured in pipeline", jobName)
 		}
 
-		build, err := job.CreateBuild()
+		build, err := job.CreateBuild("some-user")
```

Comment:
It would be nice to convert more of the tests to use this new `dbtest` package so there doesn't have to be similar churn across `db/*_test.go` in the future, but def not a blocker for the PR (that's a lot of work!).

Reply:
Thanks for the info. However I'd consider that is out of  scope of this PR.